src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -204,6 +215,19 @@ public function addSearchField(FieldDescriptorInterface $fieldDescriptor)
     public function search($search)
     {
         $this->search = $search;
+
+        return $this;
+    }
+
+    public function filter($filters)
+    {
+        foreach ($filters as $fieldName => $options) {
+            $fieldDescriptor = $this->fieldDescriptors[$fieldName];
+            $this->filterTypeRegistry->getFilterType($fieldDescriptor->getMetadata()->getFilterType())
```

Comment:
whould we throw an explicit exception if the `getFilterType()` method returns null?

Reply:
Yes, `getFilterType` will throw an exception